src/domains/ecommerce-app/env/weu-prod/terraform.tfvars

```diff
@@ -47,3 +47,10 @@ ecommerce_vpos_psps_list = "BNLIITRR,BCITITMM,UNCRITMM,BPPIITRRXXX,PPAYITR1XXX,B
 ecommerce_npg_psps_list  = ""
 
 dns_zone_checkout = "checkout"
+
+# BEGIN Variables needed for PM <-> eCommerce compatibility layer for IO app
+
+pagopa_proxy_basepath = "CHANGEME"
```

Comment:
are this variables not needed for dev and uat envs?

Reply:
Yep, left from refactoring in this PR, removing

Reply:
Fixed in 1183c80b